charts/stable/homepage/Chart.yaml

```
@@ -33,4 +33,4 @@ sources:
   - https://github.com/truecharts/charts/tree/master/charts/stable/homepage
   - https://ghcr.io/gethomepage/homepage
 type: application
-version: 7.2.0
+version: 8.0.0
```

```suggestion
version: 7.2.1
```

I'm going to flag this "non-breaking", has the templating automates what otherwise would be mandatory configuration.